ci/pipelines/noble-stemcell.md

```diff
@@ -0,0 +1,17 @@
+# noble-stemcell
+
+Test cf-d on the Ubuntu Noble stemcell.
+
+## Triggers
+
+This pipeline is automatically triggered when new Noble stemcells are published, or when a cf-d commit passes through the cf-deployment CI to be promoted to the `release-candidate` branch.
```

Comment:
Maybe mention the stemcell repository that we use as trigger: https://bosh.io/stemcells/#ubuntu-noble

Reply:
Done.